alot/defaults/alot.rc.spec

```diff
@@ -351,6 +351,14 @@ thread_focus_linewise = boolean(default=True)
         #           1.0, please move to `all`, `none`, or `trusted`.
         encrypt_by_default = option('all', 'none', 'trusted', 'True', 'False', 'true', 'false', 'Yes', 'No', 'yes', 'no', '1', '0', default='none')
 
+        # If this is true when encrypting a message it will also be encrypted
+        # with the key defined for this account.
+        #
+        # .. warning::
+        #   
+        #    Before 0.6 this was controlled via gpg.conf.
```

Comment:
shouldn't this be 0.7?

Reply:
Since 0.6 gpg.conf doesn't work for this. It's true that I guess 0.7 will be the first version with `encrypt_to_self`.

Reply:
ok i see. fine then :)